awesomerc.lua

```diff
@@ -175,6 +174,14 @@ awful.screen.connect_for_each_screen(function(s)
         end
         gears.wallpaper.maximized(wallpaper, s, true)
     end
+end
+
+-- Re-set wallpaper when a screen's geometry changes (e.g. different resolution)
+screen.connect_signal("property::geometry", set_wallpaper)
```

Comment:
Does removing a screen count as a screen change? You definitely don't want to set a wallpaper there (quite a wasted effort and it might actually cause "the wrong thing" to happen, if some other screen takes the place of the old one). So I'm not really sure about the semantics of `awful.screen.connect_for_each_screen_change`.